tests/Geppetto/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "geppettov2",
+  "version": "1.1.0",
+  "description": "GeppettoV2 is our new version of test Automation Framework for Prestashop",
+  "main": "index.js",
+  "scripts": {
+    "mocha-test-puppeteer": "node_modules/mocha/bin/mocha test/campaigns/clients/take_screenshot.js",
+    "specific-test": "node_modules/mocha/bin/mocha test/campaigns/suite/$TEST_PATH"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mbadrani/geppettov2.git"
+  },
+  "keywords": [
+    "E2E",
+    "test",
+    "automation",
+    "puppetteer"
+  ],
+  "author": "mbadrani",
+  "license": "ISC",
```

Comment:
Not sure it is the good license to use. Ping @ttoine

Reply:
fixed in my next PR thanks